### DISCUSSION TITLE
Airflow Genie Operators

### DISCUSSION THREAD

**Discussion by peedee03** [Netflix/genie#1106] [Q&A]
Hi Folks,

Can someone help me with how easy is it to integrate Airflow, Genie and EMR? Couldn't find any documentation around this - any pointers would be helpful here. Thanks in advance.

**Comment by tgianos** [comment]
Hey @peedee03,

I'm not super familiar with Airflow but you could create operator wrapping the Genie client. As for the Genie/EMR setup the [older release docs](https://netflix.github.io/genie/docs/3.3.19/reference/#_netflix_example) reference how EMR used to be used at Netflix.
